app/assets/stylesheets/_colors.scss

```diff
@@ -0,0 +1,3 @@
+$color-red: #F44336;
```

Comment:
Small letters. Don't use the word `color` in front. Also, try to use more descriptive words. https://html-color-codes.info/color-names/ this can help :)